lib/utils/fs/fileExistsSync.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const fse = require('./fse');
+
+function fileExistsSync(filePath) {
```

Comment:
You could use `this.serverless.utils.fileExistsSync` so this code can be removed.